NMS-8875: Topology is slow due to geocoding failures

mvrueden opened the pull request:
* JIRA: https://issues.opennms.org/browse/NMS-8875

j-white commented on features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/info/CoordinateResolver.java:
```diff
@@ -98,22 +99,21 @@ Result resolve(Collection<Integer> nodeIds) {
+        for (OnmsNode eachNode : nodesWithAddress) {
+            final String addressString = geoLocation(eachNode).asAddressString();
+            try {
+                org.opennms.features.geocoder.Coordinates coordinates = geocoderService.getCoordinates(addressString);
```
The timeout will help reduce page load times, but won't eliminate the slowness all together, since we are still blocking the page while attempting to geocode each node in series.

Further, if this geocoding fails during the initial page load, it will still be reattempted every-time the view changes.

I think we need a better way of handling this.

mvrueden commented:
I added the possibility to disable the resolution of geolocations based on the address string if no long/lat information is available. In addition I documented the properties available to configure the Topology Map (and removed properties not useful to configure at all).
